### PR TITLE
Tags updates, phase 1

### PR DESCRIPTION
**Pull request by dvalinrh**

### CODE REVIEW

File: ansible_roles/roles/aws_create/files/tf/main.tf [hunk 1]
```diff
@@ -7,18 +7,3 @@ terraform {
   }
   required_version = ">= 0.14.9"
 }
-
```
Comment (dvalinrh) [inline ansible_roles/roles/aws_create/files/tf/main.tf:10]: This is now dynamically done, based on the contents of config/tags.conf.

File: ansible_roles/roles/aws_create/files/tf/vars.tf [hunk 2]
```diff
@@ -40,26 +40,6 @@ variable "user" {
   default = "none"
 }
 
-variable "owner" {
```
Comment (dvalinrh) [inline ansible_roles/roles/aws_create/files/tf/vars.tf:43]: This is now generated based on the contents of config/tags.conf.

File: ansible_roles/roles/aws_create/tasks/aws_instance_create.yml [hunk 3]
```diff
@@ -243,13 +219,3 @@
   when:
    - config_info.cloud_numb_networks != 0
    - config_info.os_vendor == "amazon"
-
```
Comment (dvalinrh) [inline ansible_roles/roles/aws_create/tasks/aws_instance_create.yml:246]: Metadata is now pulled from different location.  We were only getting the metadata for one system when we had 2 for networks.

File: ansible_roles/roles/aws_create/templates/tfvars.j2 [hunk 4]
```diff
@@ -17,12 +17,6 @@ security_group = "{{ dyn_data.aws_uname }}"
 
 key_name = "{{ dyn_data.aws_uname }}"
 
-user = "{{ config_info.user_running }}"
```
Comment (dvalinrh) [inline ansible_roles/roles/aws_create/templates/tfvars.j2:20]: Removed, this is generated for us now.

File: ansible_roles/roles/azure_create/files/tf/main_net.tf [hunk 5]
```diff
@@ -10,28 +10,12 @@ provider "azurerm" {
   features {}
 }
 
-# Create resource group
```
Comment (dvalinrh) [inline ansible_roles/roles/azure_create/files/tf/main_net.tf:13]: Removed this is generated for us.   Note at this time, only the overall resource gets tagged, opening a Jiri issue to do all the resources, but will require some investigation and more rework.

File: ansible_roles/roles/azure_create/files/tf/main_net.tf [hunk 6]
```diff
@@ -61,10 +45,6 @@ resource "azurerm_public_ip" "publicip" {
     location             = var.location
     resource_group_name  = azurerm_resource_group.resource_group.name
     allocation_method    = "Dynamic"
-
```
Comment (dvalinrh) [inline ansible_roles/roles/azure_create/files/tf/main_net.tf:64]: Remove all tags on sub resource level at this time, misleading at best.

File: bin/kick_off.sh [hunk 7]
```diff
@@ -89,7 +89,7 @@ spot_recover=1
 create_attempts=5
 remove_dirs=0
 ssh_key_file=""
-while getopts "a:c:d:f:s:r:S:t:" o; do
```
Comment (dvalinrh) [inline bin/kick_off.sh:92]: Fix option bug, -r does not take an option

**Review by sousinha97 — APPROVED** [review]

**Pull request merged**